core/src/test/java/io/questdb/test/griffin/ExpressionParserTest.java

```diff
@@ -358,6 +358,69 @@ public void testCaseWithDanglingCast() {
         );
     }
 
+    @Test
+    public void testCountDistinctRewrites() throws SqlException {
```

Comment:
In Postgres `count(distinct ...)`  supports multiple columns, e.g. `count(distinct (col1, col2))`. We should add a few tests to verify that we give a meaningful error in such a case.